src/frontend/components/Accordion/Accordion.js

```diff
@@ -0,0 +1,27 @@
+import "./Accordion.css";
+import React from "react";
+import { useState } from "react";
```

Comment:
You could import `useState` on the 2nd line itself with `React` as follows:
`import React, { useState } from "react";`

Reply:
yes I'll. vs codes auto things ;)